src/components/bulkEdits/BulkEdits.tsx

```diff
@@ -0,0 +1,322 @@
+        })
+    }
+
+    renderBulkEditHeader = () => {
+        return (<div className="page-header brdr-btm pl-20">
+            <div className="page-header__title flex left fs-16 pt-16 pb-16 "> Run Scripts
+                <Tippy className="default-tt " arrow={false} placement="top" content={
+                    <span style={{ display: "block", width: "66px" }}> Learn more </span>}>
+                    <Question className="icon-dim-20 ml-16 cursor" />
+                </Tippy>
+            </div>
+        </div>)
+    }
+
+    renderBulkHeaderDescription = () => {
+        return (
+            <div className="deployment-group-list-page">
+                <div className="bulk-desciription flex left pt-10 pb-10 pl-20 pr-20 cn-9" >
+                    <Question className="icon-dim-16 mr-13" style={{ stroke: "#664bee" }} />
+                    <div>Run scripts to bulk edit configurations for multiple devtron components.
+                      <a className="learn-more__href" href={DOCUMENTATION.APP_CREATE_ENVIRONMENT_OVERRIDE} rel="noreferrer noopener" target="_blank"> Learn more</a>
+                    </div>
+                    <Close style={{ margin: "auto", marginRight: "0" }} className="icon-dim-20 cursor" onClick={() => this.setState({ showHeaderDescription: false })} />
+                </div>
+            </div>
+        )
+    }
+
+    handleRunButton = () => {
+        this.setState({
+            view: ViewType.LOADING
+        })
+
+        let configJson: any = {};
+        try {
+            configJson = yamlJsParser.parse(this.state.codeEditorPayload)
+        }
+        catch (error) {
+            //Invalid YAML, couldn't be parsed to JSON. Show error toast
+            toast.error("Invalid Yaml");
+            this.setState({ view: ViewType.FORM });
+            return;
+        }
+
+        let payload = configJson
+
+        updateBulkList(payload).then((response) => {
+            let output = response.result;
+            this.setState({
+                view: ViewType.FORM,
+                outputList: output,
+                showObjectsOutputDrawer: true
+            })
+        })
+            .catch((error) => {
+                showError(error);
+                this.setState({ view: ViewType.FORM });
+            })
+    }
+
+    handleShowImpactedObjectButton = () => {
+        let configJson: any = {};
+        try {
+            configJson = yamlJsParser.parse(this.state.codeEditorPayload)
+        }
+        catch (error) {
+            //Invalid YAML, couldn't be parsed to JSON. Show error toast
+            toast.error("Invalid Yaml");
+            this.setState({ view: ViewType.FORM });
+            return;
+        }
+
+        let payload = configJson
+
+        updateImpactedObjectsList(payload).then((response) => {
+            let result = response.result.map((elm) => elm.appName)
+            this.setState({
+                view: ViewType.FORM,
+                impactedObjectList: result,
```

Comment:
name should be impactedObjects, not impactedObjectList, never include datastructure in name

Reply:
fixed